src/scan.c

```diff
@@ -180,6 +181,7 @@ static void scanLabel(const char **iter) {
         scanstackPushLabel(start, end - start, STOP);
         scanstackPush(PUSH1);
     }
+    scanWaste(iter);
```

Comment:
this fixed an issue where we would reach the end of the file if it ends in a label. Technically allowed, tho unusual.